src/main/java/az/edu/turing/modul1/Task1A.java

```diff
@@ -0,0 +1,60 @@
+package az.edu.turing.modul1;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Task1A {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("-----------LET THE GAME BEGIN---------------\n");
+
+
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+        //  int randomNumber = (int) (Math.random() * 100);
+
+
+        System.out.print("Enter the name: ");
+        String name = sc.nextLine();
+
+        System.out.println("Enter the number you want to,between 0 and 100");
+        int number = sc.nextInt();
+        sc.nextLine();
```

Comment:
You can write this inside loop

Reply:
I can not write number inside loop, because my condistion due number.(while number!=randomNumber) 
I made some changes in the code.